plonk-napi/src/pasta_fp_plonk_index.rs

```diff
@@ -29,15 +22,19 @@ use std::{
     io::{BufReader, BufWriter, Cursor, Seek, SeekFrom::Start},
     sync::Arc,
 };
-pub struct NapiPastaFpPlonkIndex(pub Box<ProverIndex<GAffine, OpeningProof<GAffine>>>);
+
+use crate::tables::{
+    lookup_table_fp_from_js, runtime_table_cfg_fp_from_js, JsLookupTableFp, JsRuntimeTableCfgFp,
+};
+pub struct WasmPastaFpPlonkIndex(pub Box<ProverIndex<GAffine, OpeningProof<GAffine>>>);
 
 #[derive(Serialize, Deserialize)]
 struct SerializedProverIndex {
     prover_index: Vec<u8>,
     srs: Vec<u8>,
 }
 
-impl NapiPastaFpPlonkIndex {
```

Comment:
Mmmm doesn't `#[napi(js_name = "WasmPastaFpPlonkIndex")]` work here? I was trying to rename at the napi level these functions so they are prefixed Napi inside plonk-napi, but then read as Wasm- from outside.